CMakeLists.txt

```diff
@@ -49,6 +49,17 @@ find_package(CURL REQUIRED)
 find_package(nlohmann_json "3.6.1" REQUIRED)
 find_package(OpenSSL REQUIRED COMPONENTS Crypto SSL)
 
+
+include(FetchContent)
+FetchContent_Declare(
+  jwt
+  GIT_REPOSITORY https://github.com/Thalhammer/jwt-cpp.git
```

Comment:
This appears to be a new dependency that is not provided by distribution package repositories. Any dependency that isn't or can't be provided by the distributions should be packaged as an externals package. Pulling things down from the internet at pre-build configure time is a no-go; at no point during the build process should internet access be required.